### PR TITLE
[server] add organizationId to old workspaces

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

There are still some workspaces without an organizationId for inactive users. This job migartes the workspaces and sets the orgId.

## Related Issue(s)

Fixes #

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: components/server/src/jobs/org-only-migration-job.ts [hunk 1]
```diff
@@ -50,11 +58,11 @@ export class OrgOnlyMigrationJob implements Job<MigrationState> {
     }
 
     public async run(state?: MigrationState): Promise<MigrationState> {
```
Comment (easyCZ) [inline components/server/src/jobs/org-only-migration-job.ts:60]: What feeds the state back into the migration? Given we select only ones which do not have Org ID, and then we set the Org ID, it seems we shouldn't really need the state at all, as the idea is we repeatedly reduce the size of the set we query such that eventually it has no results

Comment (svenefftinge) [reply components/server/src/jobs/org-only-migration-job.ts:60]: We query them in chunks of 3000 each time the job runs continuing with workspaces newer that the newest from the last run,

Comment (easyCZ) [reply components/server/src/jobs/org-only-migration-job.ts:60]: But that's true irrespective of the "last time" right? Say we have 10k records to update, we take 3k, update them and on subsequent run, there are only 7k entries, out of which we take 3k again. That means we don't need the state for this

Comment (svenefftinge) [reply components/server/src/jobs/org-only-migration-job.ts:60]: Ah got it. You are right there should be no workspace older already. Do you insist on changing this (given it is a one-off migration)?

**Review by easyCZ — APPROVED** [review]

**Pull request merged**